blas64/blas64.go

```diff
@@ -302,43 +309,43 @@ func Sbmv(alpha float64, a SymmetricBand, x Vector, beta float64, y Vector) {
 
 // Spmv performs
 //    y = alpha * A * x + beta * y,
-// where A is an n×n symmetric matrix in packed format, x and y are vectors
+// where A is an n×n symmetric matrix in packed format, x and y are vectors,
 // and alpha and beta are scalars.
 func Spmv(alpha float64, a SymmetricPacked, x Vector, beta float64, y Vector) {
 	blas64.Dspmv(a.Uplo, a.N, alpha, a.Data, x.Data, x.Inc, beta, y.Data, y.Inc)
 }
 
-// Ger performs the rank-one operation
-//  A += alpha * x * y^T
+// Ger performs a rank-1 update
+//  A = alpha * x * y^T + A,
```

Comment:
Two reasons. I take these descriptions as mathematical rather than Go-like, therefore I wanted to avoid `+=`. Also, this is more in line with functions like `Symm` that multiply the destination with a scalar.